server/graphql/v1/mutations/orders.js

```diff
@@ -786,14 +786,11 @@ export async function refundTransaction(_, args, req) {
   const HostCollectiveId = isHost ? collective.id : await collective.getHostCollectiveId();
 
   // 1. Verify user permission. User must be either
-  //   a. Admin of the collective that received the donation - disabled for now
+  //   a. Admin of the collective that received the donation
   //   b. Admin of the Host Collective that received the donation
   //   c. Admin of opencollective.com/opencollective
 
-  if (
-    !req.remoteUser.isAdmin(HostCollectiveId) &&
-    !req.remoteUser.isRoot() /* && !req.remoteUser.isAdmin(collective.id) */
-  ) {
+  if (!req.remoteUser.isAdmin(HostCollectiveId) && !req.remoteUser.isRoot() && !req.remoteUser.isAdmin(collective.id)) {
```

Comment:
Shouldn't we introduce the logic about 30 days policy here?

Reply:
Yes! I hadn't quite gotten around to that yet, but thanks for pointing out where it should go. I wasn't sure.

Reply:
This is the function for all refunds as well as now being used for rejects.

Do we want to enforce the 30 day policy on all these? cc @znarf @kewitz @piamancini

Reply:
@Betree What's stopping us from implementing Paypal refunds? Have we just not got around to it or is there a blocker?